tests/core/full_node/test_performance.py

```diff
@@ -119,7 +119,7 @@ async def test_full_block_performance(
         pr = cProfile.Profile()
         pr.enable()
```

Comment:
I would think the profilers slow this down a bit. We should probably remove them.